server.rb

```diff
@@ -54,6 +55,13 @@
   JSON.generate Hash.from_xml(response.body)
 end
 
+get '/popular-content' do
+  content_type :json
+
+  content = Content.new()
+  content.most_popular_govuk_pages
+end
+
```

Comment:
Nice! I love this use of an internal route - that calls an internal class - as a substitute for calling an external API directly. Very clean 👏